ReactiveCocoa.podspec

```diff
@@ -17,10 +17,11 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/ReactiveCocoa/ReactiveCocoa.git", :tag => "#{s.version}" }
   s.source_files = "ReactiveCocoa/*.{swift,h,m}"
   s.private_header_files = "ReactiveCocoa/RACObjCRuntimeUtilities.h"
-  s.osx.source_files = "ReactiveCocoa/AppKit/*.{swift}"
-  s.ios.source_files = "ReactiveCocoa/UIKit/*.{swift}"
-  s.tvos.source_files = "ReactiveCocoa/UIKit/*.{swift}"
+  s.osx.source_files = "ReactiveCocoa/AppKit/*.{swift}", "ReactiveCocoa/Shared/*.{swift}"
+  s.ios.source_files = "ReactiveCocoa/UIKit/*.{swift}", "ReactiveCocoa/Shared/*.{swift}"
+  s.tvos.source_files = "ReactiveCocoa/UIKit/*.{swift}", "ReactiveCocoa/Shared/*.{swift}"
   s.tvos.exclude_files = "ReactiveCocoa/UIKit/*{UIDatePicker,UISwitch}*"
+  s.watchos.exclude_files = "ReactiveCocoa/Shared/*.{swift}"
```

Comment:
If you don't add `"ReactiveCocoa/Shared/*.{swift}"` to `s.source_files`, this isn't needed since the files are not part of watchOS source files.